packages/x-date-pickers/src/CalendarPicker/DayPicker.tsx

```diff
@@ -62,7 +62,7 @@ export interface DayPickerProps<TDate> extends ExportedDayPickerProps<TDate> {
   TransitionProps?: Partial<SlideTransitionProps>;
 }
 
-const weeksContainerHeight = (DAY_SIZE + DAY_MARGIN * 4) * 6;
+const weeksContainerHeight = (DAY_SIZE + DAY_MARGIN * 2) * 6;
```

Comment:
No idea why we were adding 4 time the margin, we only to add it one time for margin-top and one time for margin-bottom

Reply:
It seems that margin top and margin bottom are merged, leading to only `2px` margin between weeks

Then the correct height should be

```suggestion
const weeksContainerHeight = (DAY_SIZE + DAY_MARGIN) * 6 + DAY_MARGIN;
```

Or the margin should be modified to put `4px` beween weeks and get a total of `40px` as explained in the [specification](https://material.io/components/date-pickers#specs)

Reply:
The size of the day between range and non-range is weird
I think I'll clean that to have a better vision of the spacing.